specification/containerservice/resource-manager/Microsoft.ContainerService/stable/2021-07-01/managedClusters.json

```
@@ -4183,6 +4186,28 @@
       "title": "The max price (in US Dollars) you are willing to pay for spot instances. Possible values are any decimal value greater than zero or -1 which indicates default price to be up-to on-demand.",
       "description": "Possible values are any decimal value greater than zero or -1 which indicates the willingness to pay any on-demand price. For more details on spot pricing, see [spot VMs pricing](https://docs.microsoft.com/azure/virtual-machines/spot-vms#pricing)"
     },
+    "ScaleDownMode": {
+      "type": "string",
+      "enum": [
+        "Delete",
+        "Deallocate"
+      ],
+      "x-ms-enum": {
+        "name": "ScaleDownMode",
+        "modelAsString": true,
+        "values": [
+          {
+            "value": "Delete",
+            "description": "Delete mode wil create new instances during scale up and remove instances during scale down."
```

```suggestion
            "description": "Create new instances during scale up and remove instances during scale down."
```

Repeating the value at the beginning of the description is a Go thing but (IMO) reads weird in most contexts